provider/ec2/ebs_test.go

```diff
@@ -813,6 +854,18 @@ func (s *ebsSuite) TestCreateVolumesErrors(c *gc.C) {
 			Attachment: &attachmentParams,
 		},
 		err: `IOPS specified, but volume type is "sc1"`,
+	}, {
+		params: storage.VolumeParams{
+			Tag:      volume0,
+			Size:     10000,
+			Provider: ec2.EBS_ProviderType,
+			Attributes: map[string]interface{}{
+				"volume-type": "gp2",
+				"throughput":  "30",
+			},
+			Attachment: &attachmentParams,
+		},
+		err: `"throughput" specified but volume type is "gp2"`,
```

Comment:
Would this error message be a bit clearer as `"throughput" cannot be specified when volume type is "gp2"`?